project1/application.py

```diff
@@ -80,13 +84,40 @@ def authenticate():
     if member is not None:
         if ((member.password == password and member.email == email) and member.name == name):
             session['username'] = request.form.get("email-id")
-            return redirect(url_for('index'))
+            return render_template("user.html")
         else:
             return render_template("registration.html", message = "Invalid credentials!")
     else:
         return render_template("registration.html", message = "Account does not exists..Please register!! ")
 
 
+@app.route('/Search', methods=["GET","POST"])
+def search():
+    
+    if(request.method == "POST"):
+        book_search = request.form.get("search")
+        
+        if request.form.get("isbnsearch") == "option1":
+            print(book_search +" like  "+"isbnsearch")
+            s = Books.query.filter(Books.isbn.like('%'+ book_search +'%')).all()
+            print(s)
+            return render_template("Books.html", Books = s)
+
+        elif request.form.get("titlesearch") == "option2":
+            print(book_search +" like   "+"titlesearch")
+            s = db1.session.query(Books).filter((Books.tittle.like('%'+ book_search +'%')))
```

Comment:
I see redundant lines can you reduce it and update.

Reply:
ok mam sure I will update it

Reply:
is  this got updated?

Reply:
yes mam, done

Reply:
good.Still 3 more lines can be reduced.

Reply:
Done mam